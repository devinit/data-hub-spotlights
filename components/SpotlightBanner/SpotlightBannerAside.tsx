import React, { FunctionComponent } from 'react';

interface SpotlightBannerAsideProps {
  className: string;
}

const SpotlightBannerAside: FunctionComponent<SpotlightBannerAsideProps> = ({ children, className }) => (
  <div className={className}>{children}</div>
);

export { SpotlightBannerAside };
