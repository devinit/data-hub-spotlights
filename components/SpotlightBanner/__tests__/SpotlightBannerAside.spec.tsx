/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { SpotlightBannerAside } from '../SpotlightBannerAside';

describe('SpotlightBannerAside', () => {
  test('renders correctly', () => {
    const renderer = TestRenderer.create(<SpotlightBannerAside />).toJSON();

    expect(renderer).toMatchSnapshot();
  });

  test('renders with children correctly', () => {
    const renderer = TestRenderer.create(
      <SpotlightBannerAside>
        <div>My Child</div>
      </SpotlightBannerAside>
    ).toJSON();

    expect(renderer).toMatchSnapshot();
  });
});
