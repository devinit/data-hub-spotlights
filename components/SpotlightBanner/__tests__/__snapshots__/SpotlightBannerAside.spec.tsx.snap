// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpotlightBannerAside renders correctly 1`] = `<div />`;

exports[`SpotlightBannerAside renders with children correctly 1`] = `
<div>
  <div>
    My Child
  </div>
</div>
`;
